infrastructure-playbooks/shrink-osd.yml

```diff
@@ -51,11 +55,11 @@
            -e osd_to_kill=0,1,2,3 argument."
       when: osd_to_kill is not defined
 
-    - import_role:
-        name: ceph-defaults
-        private: false
+  roles:
```

Comment:
`import_role` was correct

Reply:
@leseb this is confusing because I definitely didn't change this roles/import_roles. unless... i think i did not rebase after this: 3f62fc585f60fcaecbc783316a99cd8314aab062 -- ok i'll fix this.